.editorconfig

```diff
@@ -325,6 +325,7 @@ ij_xml_line_comment_at_first_column = true
 
 [{*.yml.example,*.yml,*.yaml}]
 indent_size = 2
+insert_final_newline = true
```

Comment:
Totally agree, and that's why I was asking for people against changing the rule in `[*]`. It may have been put there for a reason. I'll probably change it at some point.